Description: remove transifex optional dependency, install npm locally
Author: Samuel MARTIN MORO <samuel@peerio.com>
Origin: vendor
Forwarded: not-needed
Last-Update: 2015-11-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile	2015-11-16 21:43:28.000000000 -0500
+++ b/Makefile	2015-11-18 14:24:58.796980934 -0500
@@ -39,12 +39,8 @@
 	if ! test -x /usr/bin/npm -o -x /usr/local/bin/npm; then \
 	    curl -k -L https://npmjs.org/install.sh | sudo sh; \
 	fi
-	if ! test -x /usr/bin/tx; then \
-	    pip list 2>&1 | grep ^transifex-client >/dev/null || sudo pip install transifex-client; \
-	fi
 	if ! test -d build; then \
-	    npm -g ls 2>&1 | grep ' nw@' >/dev/null || sudo npm install -g nw; \
-	    test -d node_modules || npm install; \
+	    test -x node_modules/.bin/nw || npm install; \
 	    test -d application/node_modules || cd application && npm install; \
 	fi
 
